docs/src/app/new/(content)/components/dialog/UnstyledDialogIntroduction/index.ts

```diff
@@ -0,0 +1,2 @@
+export { default as CssModules } from './css-modules';
+export { default as Tailwind } from './tailwind';
```

Comment:
A small price to pay for static imports, we'll need to do copy around an `index.ts` file for each demo folder so that it can be imported into the MDX file

Reply:
All demos will have the same set of CSS options so this should be a non-issue